test/kickstart-templates/includes/post-firewall.cfg

```diff
@@ -0,0 +1,10 @@
+dnf install -y firewalld
+firewall-offline-cmd --zone=trusted --add-source=10.42.0.0/16
+firewall-offline-cmd --zone=trusted --add-source=169.254.169.1
+firewall-offline-cmd --zone=public --add-port=80/tcp
+firewall-offline-cmd --zone=public --add-port=443/tcp
+firewall-offline-cmd --zone=public --add-port=5353/udp
+firewall-offline-cmd --zone=public --add-port=6443/tcp
+firewall-offline-cmd --zone=public --add-port=30000-32767/tcp
+firewall-offline-cmd --zone=public --add-port=30000-32767/udp
+firewall-offline-cmd --zone=public --add-service=mdns
```

Comment:
There's a function in scenario.sh to set up the firewall on a running host. We also have these settings in blueprints, which of course aren't used for CentOS, but I would at least like to avoid adding a 3rd location with the same settings.